### PR TITLE
exiqgrep: add page

### PR DESCRIPTION
**Pull request by ovigeek**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by ovigeek** [comment]
> The [build](https://github.com/tldr-pages/tldr/actions/runs/6949757259) for this PR failed with the following error(s):
> 
> ```
> pages/linux/exiqgrep.md:22: TLDR005 Example descriptions should end in a colon with no trailing characters
> ```
> 
> Please fix the error(s) and push again.

Fixed line 22

File: pages/linux/exiqgrep.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# exiqgrep
+
+> The **exiqgrep** utility is a Perl script which offers possibilities to grep in the exim queue output.
```
Comment (kbdharun) [inline pages/linux/exiqgrep.md:3]: Maybe we can update the description a bit.

```suggestion
> The `exiqgrep` utility is a Perl script offering possibilities to `grep` in the Exim queue output.
```

Or

```suggestion
> Perl script offering possibilities to `grep` in the Exim queue output.
```

File: pages/linux/exiqgrep.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`exiqgrep -i -f '<{{email@somedomain.com}}>'`
+
+- Match the recipient address:
+
+`exiqgrep -r '{{email@somedomain.com}}'`
+
+- Remove from queue all messages, matching sender address:
+
+`exiqgrep -i -f '<{{email@somedomain.com}}>' | xargs exim -Mrm`
```
Comment (kbdharun) [inline pages/linux/exiqgrep.md:20]:
```suggestion
`exiqgrep -f '<{{email@somedomain.com}}>'`

- Match the sender address, and display message IDs only:

`exiqgrep -i -f '<{{email@somedomain.com}}>'`

- Match the recipient address:

`exiqgrep -r '{{email@somedomain.com}}'`

- Remove all messages matching the sender address from the queue:

`exiqgrep -i -f '<{{email@somedomain.com}}>' | xargs exim -Mrm`
```

File: pages/linux/exiqgrep.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Test for bounced messages:
+
+`exiqgrep -f '^<>$'`
+
+- Display bounced messages count:
```
Comment (kbdharun) [inline pages/linux/exiqgrep.md:26]:
```suggestion
- Display the count of bounced messages:
```

**Review by kbdharun — COMMENTED** [review]
Welcome to tldr and thanks for your contribution. I have some suggestions for this page.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**